README.md

```diff
@@ -44,82 +43,54 @@ The middleware is a caching and reporting layer which sits in front of the nodes
 
 The architecture of the app is explained [here](docs/architecture.md).
 
-## Prerequisites
-
-Ensure that you have [Elixir](https://elixir-lang.org/install.html) installed, using Erlang 22 or newer.
 
 ## Setup
 
 Firstly, clone the middleware repo:
 
 `git clone https://github.com/aeternity/ae_mdw && cd ae_mdw`
 
-For syncing on testnet, change ./docker/aeternity.yaml network_id to `ae_uat`.
-
-Following there are two options depending on which block height you want to continue syncing:
-
-### A) Sync from genesis block
+Before running it it's recommeded to use a Node database snapshot for faster syncing.
 
-1. Simply run the middleware with `docker-compose up`
+### Database Snapshot
 
-Next time you run the same command it will continue from the last height synced.
+1. Download the one of the full backups from https://downloads.aeternity.io
+2. Create a 'data' directory under the root repo dir and extract the backup to it.
```

Comment:
Perhaps we should mention that it's the `mnesia` directory that should be extracted into the local `data/mnesia` directory

Reply:
The backup contain the `mnesia` directory. Added a mention as a result.